js/bartender.js

```diff
@@ -1,86 +1,192 @@
-
-var wineColor = "";
-var wineFlavor = "";
-var wineWeight = "";
-var selection = "";
-
-$('.wine-color').hide();
-$('.wine-flavor').hide();
-$('.wine-weight').hide();
-$('.deliver-blend').hide();
-
+var wineType = '';
+var wineColor = '';
+var wineFlavor = '';
+var wineWeight = '';
+var selection = '';
+var blend = '';
+var min = 0;
+
+// collecting a customer's preferences
 var CustPreferences = function(preferences) {
     this.preferences = preferences;
 };
 
 CustPreferences.prototype.addPreference = function(selection) {
     this.preferences.push(selection);
-    //console.log('this addPreferences.preferences are ' + this.preferences);
 };
 
-var Bartender = function(name) {
-    this.name = name;
+// adding to inventory and tagging it
+var Inventory = function() {
+    this.cellar = [];
 };
 
-Bartender.prototype.newBlend = function(customerSelection)  {
-    // logic to create and name a drink
-    if(customerSelection.includes("red") && customerSelection.includes("sweet")) {
-            return "Six of One";
-    } else
-          //if (customerSelection.includes("red") && (customerSelection.includes("dry"))
-          //    return "A Fine Finish";
-          // else   //customer selection is white
-        if(customerSelection.includes("white") && customerSelection.includes("dry")) {
-        return "Bite Your Tongue";
-        }
+Inventory.prototype.stockWine = function(wineType, wineColor, wineFlavor, wineWeight) {
+    this.cellar.push({
+        type: wineType,
+        color: wineColor,
+        flavor: wineFlavor,
+        weight: wineWeight
+    });
 };
 
-$(document).ready( function() {
-
-      // get wine color preference
-    $('.wine-color').prepend("<p>Do you prefer red wine or white wine?</p>");
-    $('.wine-color').show();
-    $('.wine-color').click( function(e){
-        e.preventDefault();
-        wineColor = $('input[name=wineColor]:checked').val();
-
-        var Preference = new CustPreferences([]);
+var wineInventory = new Inventory();
 
-        Preference.addPreference(wineColor);
-        $('.wine-flavor').prepend("<p>And do you like your " + wineColor + " wine sweet or dry?</p>");
-        $('.wine-flavor').show();
+// generate random number to return one of the wines in the cellar
+function getRandomArrayIndex(inventory) {
```

Comment:
Change name of this function .. the function now returns a wine object hence name is not appropriate.